Misc/NEWS.d/next/Core_and_Builtins/2025-04-06-17-31-44.gh-issue-132166.tbo9yR.rst

```diff
@@ -0,0 +1,5 @@
+Add checking if sequences ``a[alo:ahi]`` and ``b[blo:bhi]`` are the same on the
+beginning of the method find_longest_match in SequenceMatcher. For identical
+sequences there is no reason to run whole logic when simple check can be done.
+It appears to fix issue when comparing two slightly different strings ends up
+with waiting forever for the result.
```

Comment:
```suggestion
It appears to fix issue when comparing two slightly different strings ends up
with waiting forever for the result.
```

We should be more confident here. Does it fix the issue or not? if so, let's say it. Otherwise, let's not give tentative conclusions.